### PR TITLE
add my name as a developer.

### PR DESCRIPTION
**Pull request by nai-aung26**
removed merge conflict.

### CODE REVIEW

File: teamcode.html [hunk 1]
```diff
@@ -58,10 +58,6 @@ <h2>TeamCode</h2>
         office hours, and GitHub. Code will be checked into GitHub and reviewed
         by other team members who can then merge work into the main repository.
       </p>
-      <p>The TeamCode team for Fall 2019 consists of:</p>
-      <ul>
-        <li><b>Brenton Strine</b> (Advisor)</li>
-      </ul>
       <img src="media/collaboration.jpg">
```
Comment (brentonstrine) [inline teamcode.html]: Looks like you deleted some code that I wrote during your merge conflict resolution.  We need this code! Also, your change isn't in here either! Can you put back both my code and yours?

File: test.html [hunk 2]
```diff
@@ -17,6 +17,7 @@
   <link href="styles/expandingSection.css" rel="stylesheet"/>
 </head>
 <body>
+  <p></p>
   <header class="header grid-header-container">
```
Comment (brentonstrine) [inline test.html]: This empty paragraph tag should not get merged into upstream. Can you remove it please?

File: teamcode.html (hunk 1 above)
Comment (brentonstrine) [reply teamcode.html]: I think you need to come in here and manually fix this. If you look at the changes above, this PR will currently delete the list of TeamCode members. 

To manually fix this, you'll need to do the following: 
1. From **GitHub Desktop** fetch or pull from origin. (_This gets the code from GitHub to your computer_.)
2. Open `teamcode.html` in Atom
3. Find the section above the ` image`. Looks like it's line 63.
4. Add back both the old code and your new code. Should look like this:
```
      <p>The TeamCode team for Fall 2019 consists of:</p>	
       <ul>	
        <li><b>Brenton Strine</b> (Advisor)</li>	
        <li><b>Mustafa Alsaeedi</b> (Devloper)</li>	
        <li>Nai Aung (Developer)</li>
      </ul>
```
5. Commit
6. Push to origin (_This puts the code up on GitHub._)

After that, this PR will automatically update with the updated code. 

@nai-aung26 I believe you have the same problem so you can follow these steps.

**Comment by brentonstrine** [comment]
@nai I fixed the problems on this ticket, however this is a `refcode1.0` change and I have moved the `refcode1.0` website to a new branch, no longer on `master`. So you will need to close this PR and create a new PR to the `refcode1.0` branch instead of the `master` branch.

**Comment by mathmastin** [comment]
@nai-aung26 feel free to ping me on slack if I can help out with closing this PR and opening a new one against the `refcode1.0` branch.

**Pull request closed without merging**